XIVComboVX/Config/ConfigWindow.cs

```diff
@@ -276,31 +276,31 @@ public override void Draw() {
 				ImGui.EndMenu();
 			}
 
-#if DEBUG
-			if (ImGui.BeginMenu("Debugging")) {
-
-				IPlayerCharacter? player = Service.Client.LocalPlayer;
-				if (player is null) {
-					ImGui.MenuItem("Not logged in", false);
-				}
-				else {
-					ImGui.MenuItem($"{player.Name}: {player.ClassJob.GameData!.Abbreviation.ToString().ToUpper()} ({player.ClassJob.Id})", false);
-				}
-
-				bool clickDebug = ImGui.MenuItem("Snapshot debug messages");
-				if (ImGui.IsItemHovered()) {
-					ImGui.BeginTooltip();
-					ImGui.Text("This enables a snapshot of debug messages in the dalamud log.");
-					ImGui.Text("They will appear in your log file and also in the /xllog window.");
-					ImGui.EndTooltip();
-				}
-				if (clickDebug) {
-					Service.TickLogger.EnableNextTick();
-				}
-
-				ImGui.EndMenu();
-			}
-#endif
+//#if DEBUG
+//			if (ImGui.BeginMenu("Debugging")) {
+//
+//				IPlayerCharacter? player = Service.Client.LocalPlayer;
+//				if (player is null) {
+//					ImGui.MenuItem("Not logged in", false);
+//				}
+//				else {
+//					ImGui.MenuItem($"{player.Name}: {player.ClassJob.GameData!.Abbreviation.ToString().ToUpper()} ({player.ClassJob.RowId})", false);
+//				}
+//
+//				bool clickDebug = ImGui.MenuItem("Snapshot debug messages");
+//				if (ImGui.IsItemHovered()) {
+//					ImGui.BeginTooltip();
+//					ImGui.Text("This enables a snapshot of debug messages in the dalamud log.");
+//					ImGui.Text("They will appear in your log file and also in the /xllog window.");
+//					ImGui.EndTooltip();
+//				}
+//				if (clickDebug) {
+//					Service.TickLogger.EnableNextTick();
+//				}
+//
+//				ImGui.EndMenu();
+//			}
+//#endif
```

Comment:
None of this should be commented out; it's used in debug builds for logging what the plugin is doing, as well as useful information like the ID of the player's current job. Barring Dalamud API elements getting renamed, nothing should break in here, and it looks like you already fixed that part anyway.

Reply:
Yeah, the problem here is that ClassJob doesn't contain GameData anymore, so I'll do some more research to see if there's an easier/better way of getting the abbreviation.